packages/configs/default/index.json

```diff
@@ -1,6 +1,7 @@
 {
   "bundler": "@parcel/bundler-default",
   "transforms": {
+    "*.{md,mdx}": ["@parcel/transformer-mdx"],
```

Comment:
Not sure if this should also parse standard markdown by default as normal markdown could be expected to result in html instead of JS